images/svg/gitter-brands.svg

```diff
@@ -1 +1,7 @@
-<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="gitter" class="svg-inline--fa fa-gitter fa-w-12" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512"><path fill="currentColor" d="M66.4 322.5H16V0h50.4v322.5zM166.9 76.1h-50.4V512h50.4V76.1zm100.6 0h-50.4V512h50.4V76.1zM368 76h-50.4v247H368V76z"></path></svg>
\ No newline at end of file
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="gitter" class="svg-inline--fa fa-gitter fa-w-12 " role="img" 
+xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512">
+<!--Added color to the icon using fill-->
+<path fill="#007bff" 
+ d="M66.4 322.5H16V0h50.4v322.5zM166.9 76.1h-50.4V512h50.4V76.1zm100.6 0h-50.4V512h50.4V76.1zM368 76h-50.4v247H368V76z">
+ </path>
+ </svg>
```

Comment:
Please add one blank line at the end of the file so GitHub gets rid of that ⛔ sign.

Reply:
```suggestion
 </svg>
 
```

Reply:
The error is still present.

Reply:
I did add the new line. But the symbol seems to stay at the end of the file no matter how many extra lines  I add. It was there in the original file I forked too.

Reply:
@Panquesito7  I have removed the symbol in the last commit. Can you please review it for me. This is my first PR so I might be a bit sloppy apologies for that.